chore(PPDSC-2046): review PR template (#139)
newscorp-ghfb/newskit: .github/PULL_REQUEST_TEMPLATE.md

<!---
Add any breaking change if present.
E.g:
BREAKING CHANGE: renames the foobar component's prop foo to bar
--->

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [ ] Updated relevant documentation

**I have tested manually:**
- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge

> - [ ] The screen reader reads the expected text for the tabbable elements.

Reviewer: ```suggestion
- [ ] The screen reader reads and flows through the elements as expected.
```

I would change it to this! A bit more correct, as is not just for tabbable elements! 👍
